src/ugeneui/src/plugin_viewer/PluginViewerController.cpp

```diff
@@ -45,16 +45,14 @@ PluginViewerController::PluginViewerController() {
     showServices = false;    //'true' mode is not functional anymore after service<->plugin model refactoring
     mdiWindow = NULL;
     connectStaticActions();
-
-    if (AppContext::getSettings()->getValue(PLUGIN_VIEW_SETTINGS + "isVisible", false).toBool()) {
-        createWindow();
-    }
 }
 
 PluginViewerController::~PluginViewerController() {
     AppContext::getPluginSupport()->disconnect(this);
 
-    AppContext::getSettings()->setValue(PLUGIN_VIEW_SETTINGS + "isVisible", mdiWindow != NULL);
+    // 'true' mode is not functional anymore after service<->plugin model refactoring
+    // so always save false for pluginview settings, also see UGENE-6369
+    AppContext::getSettings()->setValue(PLUGIN_VIEW_SETTINGS + "isVisible", false);
```

Comment:
Why do we save the value? Is it used anywhere? If not, lets remove this line

+ The same for the PLUGIN_VIEW_SETTINGS constant

Reply:
removed

Reply:
I think we can remove PLUGIN_VIEW_SETTINGS constant also - it is not used anymore.